cli/src/athenapdf.js

```diff
@@ -22,6 +22,8 @@ if (!process.defaultApp) {
     process.argv.unshift("--");
 }
 
+app.commandLine.appendSwitch('ignore-gpu-blacklist', 'true');
```

Comment:
I'm not too sure how I feel about this (i.e. security repercussions). It should be fine considering most use-cases, but should we perhaps protect it with a command line flag?